python/cuml/test/test_api.py

```diff
@@ -81,22 +81,10 @@ def dataset():
 def test_get_tags(model):
     # This test ensures that our estimators return the tags defined by
     # Scikit-learn and our cuML specific tags
-    # mod = models[model_name]
-    # assert hasattr('_get_tags', m)
 
-    # for tag in tags:
-    #     assert
+    assert hasattr(model, '_get_tags')
 
-    print(model)
-    if model in (cuml.tsa.auto_arima.AutoARIMA, cuml.tsa.arima.ARIMA,
-                 cuml.tsa.holtwinters.ExponentialSmoothing):
-        mod = model(cp.ones(10))
-    else:
-        mod = model()
-
-    assert hasattr(mod, '_get_tags')
-
-    model_tags = mod._get_tags()
+    model_tags = model._get_tags()
```

Comment:
```suggestion
    model_tags = model._get_tags()
    
    if (hasattr(model, "_more_tags")):
        import inspect
        assert(isinstance(inspect.getattr_static(model, "_more_tags"), staticmethod))
```
Also might want to add something like this to ensure any `_more_tags` methods are not `classmethod` or instance methods (I checked this and it works to catch both).